frontend/src/component/photo/mosaic.vue

```diff
@@ -20,106 +20,19 @@
       </v-card-title>
     </v-card>
     <v-layout row wrap class="p-results">
-      <v-flex
-          v-for="(photo, index) in photos"
-          :key="index"
-          :data-uid="photo.UID"
-          v-bind:class="{ selected: $clipboard.has(photo) }"
-          class="p-photo"
-          xs4 sm3 md2 lg1 d-flex
+      <p-photo-tile
+        v-for="(photo, index) in photos"
+        :key="index"
+        :index="index"
```

Comment:
Is this needed since we already have `key`? Does `key` have a special Vue meaning? Can we/should we access it from the child?